src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! # Substrate
+//!
+//! Substrate is a Rust framework for building blockchains in a modular and extensible way. While in
+//! itself un-opinionated, it is the main engine behind the Polkadot ecosystem.
+//!
+//! [![github]](https://github.com/paritytech/substrate/) - [![polkadot]](https://polkadot.network)
+//!
+//! This crate in itself does not contain any code and is just meant ot be a documentation hub for
+//! substrate-based crates.
+//!
+//! ## Overview
+//!
+//! Substrate approaches blockchain development with a focus on **safety**, **correctness**, and
+//! **upgradeability**.
+//!
+//! **Safety** is acquired through a use of Rust, a modern language empowering everyone to build
+//! reliable and efficient software.
+//!
+//! **Correctness** is ensured through a rich type system enforcing semantic guarantees. This is
```

Comment:
Correctness in which way? That it is easy to correctly implement some specific required runtime logic when using Substrate?

Reply:
I meant to lean into the heavy use of the type system, which is truthfully more of a FRAME feature than substrate. I will revise the messaging a bit.